framework/composer.json

```diff
@@ -54,7 +54,6 @@
     "require": {
         "php": ">=5.4.0",
         "ext-mbstring": "*",
-        "ext-mcrypt": "*",
```

Comment:
No. It's not actually an extension: http://php.net/manual/en/openssl.installation.php